flair/datasets/sequence_labeling.py

```diff
@@ -4973,6 +4973,209 @@ def _write_instances(cls, version, base_path, split, data):
+            self.base_path / "clean.test",
+        ]
+        files_exist = [path.exists() for path in file_paths]
+        self.cleanconll_base_path = flair.cache_root / "datasets" / "cleanconll"
+
+        if not all(files_exist):
+            cached_path(f"{self.label_url}/{filename}.traindev", self.base_path / "annotations_only")
+            cached_path(f"{self.label_url}/index.txt", self.base_path / "annotations_only")
+
+            cleanconll_files_exist = [
+                Path(f"{self.cleanconll_base_path}/cleanconll.{split}").exists() for split in ["train", "dev", "test"]
+            ]
+            if not all(cleanconll_files_exist):
+                # download cleanconll
+
+                clone = f"git clone https://github.com/flairNLP/CleanCoNLL.git {self.cleanconll_base_path}/CleanCoNLL"
+                os.system(clone)  # Cloning
+                cwd = os.getcwd()
+
+                os.chdir(f"{self.cleanconll_base_path}/CleanCoNLL")
+                chmod = "chmod u+x create_cleanconll_from_conll03.sh"
+                os.system(chmod)
+                create = "bash create_cleanconll_from_conll03.sh"
+
+                os.system(create)
+                os.chdir(cwd)
+
+                shutil.move(
+                    f"{self.cleanconll_base_path}/CleanCoNLL/data/cleanconll/cleanconll.train",
+                    self.cleanconll_base_path,
+                )
+                shutil.move(
+                    f"{self.cleanconll_base_path}/CleanCoNLL/data/cleanconll/cleanconll.dev", self.cleanconll_base_path
+                )
+                shutil.move(
+                    f"{self.cleanconll_base_path}/CleanCoNLL/data/cleanconll/cleanconll.test", self.cleanconll_base_path
+                )
+
+                shutil.rmtree(self.cleanconll_base_path / "CleanCoNLL")
+
+            # create dataset files from index and train/test splits
+            self.generate_data_files(
+                filename,
+            )
+
+        super().__init__(
+            data_folder=self.base_path,
+            train_file=f"{filename}.train",
+            dev_file=f"{filename}.dev",
+            test_file="clean.test",  # test set is always clean (without noise)
+            column_format={0: "text", 1: "ner"},
+            in_memory=in_memory,
+            column_delimiter="\t",
+            document_separator_token="-DOCSTART-",
+            **corpusargs,
+        )
+
+    def _set_path(self, base_path):
+        self.base_path = flair.cache_root / "datasets" / "noisebench" if not base_path else Path(base_path)
+
+    @staticmethod
+    def read_column_file(filename):
+        with open(filename, errors="replace") as file:
+            lines = file.readlines()
+            all_x = []
+            point = []
+            for line in lines:
+                if "\t" in line.strip():
+                    stripped_line = line.strip().split("\t") if "\t" in line.strip() else line.strip().split(" ")
```

Comment:
Looks like `stripped_line = line.strip().split("\t") if "\t" in line.strip()` will always be set due to the condition `if "\t" in line.strip()` in line 5076. Refactor to:
```python
if "\t" in line.strip():
    stripped_line = line.strip().split("\t")
else:
    stripped_line = line.strip().split(" ")